src/ImageSharp/Processing/Processors/Transforms/Linear/SkewProcessor.cs

```diff
@@ -30,7 +30,7 @@ public SkewProcessor(float degreesX, float degreesY, Size sourceSize)
         /// <param name="sourceSize">The source image size</param>
         public SkewProcessor(float degreesX, float degreesY, IResampler sampler, Size sourceSize)
             : this(
-                 TransformUtilities.CreateSkewMatrixDegrees(degreesX, degreesY, sourceSize),
+                 TransformUtils.CreateSkewMatrixDegrees(degreesX, degreesY, sourceSize),
```

Comment:
So we are back to the `Utils` postfix? Agreed, shorter is better here.

Reply:
Yeah less typing. Unfortunately there's a public `GeometryUtilities` class I couldn't change.

Reply:
We probably shouldn't have left that public for the 2 methods it has.

Reply:
Yeah... We shouldn't. Annoyed at myself for not spotting it.